src/cli.c

```diff
@@ -111,8 +111,9 @@ static int reset(SSL* ssl, int socket);
 static int reload(SSL* ssl, int socket);
 static int retain(SSL* ssl, int socket, char* server, char* backup_id);
 static int expunge(SSL* ssl, int socket, char* server, char* backup_id);
-static int decrypt(SSL* ssl, int socket, char* path);
-static int encrypt(SSL* ssl, int socket, char* path);
+// The encrypt function is a standard library function declared in unistd.h on macOS
+static int decrypt_data(SSL* ssl, int socket, char* path);
+static int encrypt_data(SSL* ssl, int socket, char* path);
```

Comment:
Why ?

Reply:
When compiling on macOS, a naming conflict arises with a function in `unistd.h`, resulting in a compilation error.

Reply:
Ok, fair enough